pagopa/pn-frontend
# Fix/PN-8718 - Implemented info toast for legal facts that are in Glacier

## Short description
Implemented info toast for legal facts that are in Glacier

## List of changes proposed in this pull request
- Added info message for SnackBar
- Added info type props for SnackBar

## How to test
Login as PF/PG/PA and open a notification sent more than 120days ago (es. September 2023). Try to download legalfacts from timeline and check an info toast appears.
